crates/tauri-bundler/src/bundle/windows/util.rs

```diff
@@ -70,8 +70,20 @@ pub fn download_webview2_offline_installer(base_path: &Path, arch: &str) -> crat
 pub fn download(url: &str) -> crate::Result<Vec<u8>> {
   log::info!(action = "Downloading"; "{}", url);
 
-  let agent = ureq::AgentBuilder::new().try_proxy_from_env(true).build();
-  let response = agent.get(url).call().map_err(Box::new)?;
+  let (agent, url) = match std::env::var("GITHUB_CDN") {
+    Ok(cdn) => {
+      let mut parsed_cdn = url::Url::parse(&cdn).expect("Invalid cdn");
+      parsed_cdn.set_path(url);
+
+      (ureq::AgentBuilder::new().build(), parsed_cdn.to_string())
+    }
+    Err(_) => (
+      ureq::AgentBuilder::new().try_proxy_from_env(true).build(),
```

Comment:
we can keep using `AgentBuilder` with `try_proxy_from_env` in the two cases, no need to make distinction, only `url` should be changed based on the new env var

Reply:
> we can keep using `AgentBuilder` with `try_proxy_from_env` in the two cases, no need to make distinction, only `url` should be changed based on the new env var

Once a proxy is set in the environment variables along with a CDN (still referred to as a CDN for now), some CDNs might refuse to respond. Therefore, I believe the CDN and the `try_proxy_from_env` should be handled separately. If both a proxy and a CDN are set in the environment variables, it's likely that the user intends to use the CDN rather than the proxy, and both should not be used simultaneously.

Reply:
makes sense